### DISCUSSION TITLE
Update lock files only through lock file maintenance.

### DISCUSSION THREAD

**Discussion by Zusel** [renovatebot/renovate#29097] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitHub

### Please tell us more about your question or problem

Hey,
I've set the "updateLockFiles" parameter to false and enabled "lockFileMaintenance."
What I aimed to achieve was to have the lock files updated only in a LockFileMaintenance PR/Branch and not with every single bump. Otherwise, we encounter the issue of too many bumps resulting in merge conflicts. However, when I set "updateLockFiles" to false, no "LockFileMaintenance" branch is created, even if it's enabled in the configuration. What do I need to do to ensure that the lock files are updated only in the LockFileMaintenance branch/PR while only the package.json is updated with the bumps?
Thank you

**Comment by rarkins** [comment]
If you update a `package.json` and do not update the `package-lock.json` accordingly then it's an invalid update. Why are you wanting that?

**Reply by Zusel** [reply]
Hey, thank you very much for the quick response. Our system is a bit complicated. The smaller updates are merged into our main branch and then updated via LockFileMaintenance. All other bumps (those that have not been merged) are consolidated into a separate branch, and the main branch is also merged there. We need this branch for testing. If I simply merge all bumps with the lock files, Renove gets merge conflicts on the lock files, and I have no other idea how to solve the problem except by prohibiting the bumps from updating the lock files directly and then updating the lock files manually (in an automated step) so that our test branch works with all bumps (even those that have not been merged into the main branch) and has no conflicts. I hope I was able to explain it somewhat understandably. Thanks a lot.

**Reply by rarkins** [reply, marked as answer]
Can you try setting updateLockFiles=true inside your `lockFileMaintenance` block to reverse the disablement? If it doesn't work then please create a reproduction repo
